pkg/controller/machine_safety_test.go

```diff
@@ -748,237 +743,237 @@ var _ = Describe("#machine_safety", func() {
-			false, true, zeroDuration, false, false),
-		Entry("Control APIServer: UnReachable, Target APIServer: Reachable, Inactive Timer: Inactive, Pre-Frozen: true = Post-Frozen: true",
-			false, true, zeroDuration, true, true),
-		Entry("Control APIServer: UnReachable, Target APIServer: Reachable, Inactive Timer: Started, Pre-Frozen: false = Post-Frozen: false",
-			false, true, fiveSecondsDuration, false, false),
-		Entry("Control APIServer: UnReachable, Target APIServer: Reachable, Inactive Timer: Started, Pre-Frozen: true = Post-Frozen: true",
-			false, true, fiveSecondsDuration, true, true),
-		Entry("Control APIServer: UnReachable, Target APIServer: Reachable, Inactive Timer: Elapsed, Pre-Frozen: false = Post-Frozen: true",
-			false, true, fiveMinutesDuration, false, true),
-		Entry("Control APIServer: UnReachable, Target APIServer: Reachable, Inactive Timer: Elapsed, Pre-Frozen: true = Post-Frozen: true",
-			false, true, fiveMinutesDuration, true, true),
-
-		// Both APIServers are not reachable
-		Entry("Control APIServer: UnReachable, Target APIServer: UnReachable, Inactive Timer: Inactive, Pre-Frozen: false = Post-Frozen: false",
-			false, false, zeroDuration, false, false),
-		Entry("Control APIServer: UnReachable, Target APIServer: UnReachable, Inactive Timer: Inactive, Pre-Frozen: true = Post-Frozen: true",
-			false, false, zeroDuration, true, true),
-		Entry("Control APIServer: UnReachable, Target APIServer: UnReachable, Inactive Timer: Started, Pre-Frozen: false = Post-Frozen: false",
-			false, false, fiveSecondsDuration, false, false),
-		Entry("Control APIServer: UnReachable, Target APIServer: UnReachable, Inactive Timer: Started, Pre-Frozen: true = Post-Frozen: true",
-			false, false, fiveSecondsDuration, true, true),
-		Entry("Control APIServer: UnReachable, Target APIServer: UnReachable, Inactive Timer: Elapsed, Pre-Frozen: false = Post-Frozen: true",
-			false, false, fiveMinutesDuration, false, true),
-		Entry("Control APIServer: UnReachable, Target APIServer: UnReachable, Inactive Timer: Elapsed, Pre-Frozen: true = Post-Frozen: true",
-			false, false, fiveMinutesDuration, true, true),
-	)
+	// const (
+	// 	zeroDuration        = time.Duration(0)
+	// 	fiveSecondsDuration = 5 * time.Second
+	// 	fiveMinutesDuration = 5 * time.Minute
+	// )
+	// DescribeTable("##reconcileClusterMachineSafetyAPIServer",
+	// 	func(
+	// 		controlAPIServerIsUp bool,
+	// 		targetAPIServerIsUp bool,
+	// 		apiServerInactiveDuration time.Duration,
+	// 		preMachineControllerIsFrozen bool,
+	// 		postMachineControllerFrozen bool,
+	// 	) {
+	// 		apiServerInactiveStartTime := time.Now().Add(-apiServerInactiveDuration)
+	// 		stop := make(chan struct{})
+	// 		defer close(stop)
+
+	// 		testMachine := &machinev1.Machine{
+	// 			ObjectMeta: metav1.ObjectMeta{
+	// 				Name:      "testmachine1",
+	// 				Namespace: testNamespace,
```

Comment:
any reason for commenting this? I think we can remove this as well.

Reply:
yes